<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_set_ecg</class>
 <widget class="QDialog" name="Dlg_set_ecg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>心电设置</string>
  </property>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>135</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>工频抑制</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>105</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>增         益</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_amp">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>4</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_chnl1_daolian">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>7</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_xintiaoyinliang">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_lvbomoshi">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>135</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ST开关</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_chnl2_daolian">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>通道1导联</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_cancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>165</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>导联类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>105</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>波形速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>15</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>滤波模式</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_chnl3_daolian">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>75</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>心率来源</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_ok">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>165</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>心率失常</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>195</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>起  博  器</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_xinlvlaiyuan">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_st">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>45</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>心跳音量</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_daolianleixing">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_qiboqi">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>通道2导联</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>75</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>通道3导联</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_boxingsudu">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_xinlvshichang">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_gongpinyizhi">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
